src/api/java/com/github/minecraftschurlimods/arsmagicalegacy/api/event/SpellEvent.java

```diff
@@ -20,15 +19,13 @@
      */
     private final ISpell spell;
 
-    public SpellEvent(@Nullable LivingEntity entity, ISpell spell) {
+    public SpellEvent(LivingEntity entity, ISpell spell) {
```

Comment:
Why remove @Nullable?

Reply:
Because the entity isn't nullable, it is always the entity using the item